### PR TITLE
a rough version of gameArea which contains board and videoarea

### PR DESCRIPTION
**Pull request by xiaozdezhu**
Herbert Li @herbertli  and me work out this rough version of gameArea which contains board and videoarea.
Please let us know if it's going to work.

### CODE REVIEW

**Comment by yoav-zibin** [comment]
Note that travis (the CI build) failed:

ts-loader: Using typescript@2.7.2 and /home/travis/build/yoav-zibin/NewGamePortal/tsconfig.json
Failed to compile.
/home/travis/build/yoav-zibin/NewGamePortal/src/reducers/index.test.ts
(55,7): Type '{ gamesList: { gameSpecId: string; gameName: string; screenShoot: Image; }[]; gameSpecs: { imageI...' is not assignable to type 'StoreState'.
  Property 'activeGame' is missing in type '{ gamesList: { gameSpecId: string; gameName: string; screenShoot: Image; }[]; gameSpecs: { imageI...'.

File: src/components/PlayingScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import Board from './Board';
+import VideoArea from './VideoArea';
+import { StoreState } from '../types/index';
+import { connect } from 'react-redux';
+
+const mapStateToProps = (state: StoreState) => {
+  const selectedGameId = state.activeGame;
```
Comment (yoav-zibin) [inline src/components/PlayingScreen.tsx]: There is no state.activeGame :)
You meant:
  currentMatchIndex: number; // an index in matchesList

File: src/types/index.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ export interface StoreState {
   myUser: MyUser;
 
   signals: SignalEntry[];
+  activeGame: string; // gameSpecId of selected game
```
Comment (yoav-zibin) [inline src/types/index.ts]: no need to change store :)

File: src/index.tsx [hunk 3]
```diff
@@ -17,6 +18,7 @@ ReactDOM.render(
       <BrowserRouter basename="/">
         <div>
           <Route path="/" component={App} />
+          <Route path="/board" component={PlayingScreen} />
```
Comment (apsdehal) [inline src/index.tsx]: This should be `/match/${matchId}`

**Comment by apsdehal** [comment]
Tests are failing.

**Comment by xiaozdezhu** [comment]
Hi Amanpreet, do you have time tomorrow? Herbert and me would like to ask you some questions about our next move.

**Comment by apsdehal** [comment]
Oh, I missed the last comment. I was actually out of the country. Sorry. You can come to office hours tomorrow.

**Pull request merged**